src/command_modules/azure-cli-lab/azure/cli/command_modules/lab/help.py

```diff
@@ -170,6 +170,6 @@
                   short-summary: Name of the Lab
                 - name: --name -n
                   short-summary: Name of the secret
-                - name: --value
+                - name: --secret
```

Comment:
The alias of the dest parameter 'secret' to --value is a logical (and I'd argue good) one.